config.ini

```diff
@@ -0,0 +1,23 @@
+# https://docs.python.org/3/library/configparser.html
+
+[COMMON]
+YAPI_dir = ~/YAPI
+platform =
+file_extension = .sh
+language = en
+language_dir = ${YAPI_dir}/language
+readme_path = ${COMMON:YAPI_dir}/README.md
+
+[REMOTE]
+raw_location = https://raw.githubusercontent.com/YetAnotherPackageInstaller/
+origin_location = https://github.com/YetAnotherPackageInstaller/YAPI
+branch = master
+zip = ${REMOTE:origin_location}/archive/${REMOTE:branch}.zip
+
+[PACKAGES]
+ignore = test.sh, updateYapiScripts.sh
+installed =
```

Comment:
What is the value of installed supposed to be?

Reply:
I want to track all the packages installed with yapi, but maybe it is better to do with a cache file like what i've done with scripts.bin